integrated_channels/integrated_channel/exporters/content_metadata.py

```diff
@@ -492,48 +511,44 @@ def export(self, **kwargs):
-                    # the channel specific, expected form
-                    item = self._transform_exec_ed_content(item)
-
-                    # transform the content metadata into the channel specific format
-                    transformed_item = self._transform_item(item)
-                    if key in items_create_keys:
-                        existing_record = items_to_create.get(key)
-                        existing_record.channel_metadata = transformed_item
-                        existing_record.content_title = item.get('title')
-                        existing_record.content_last_changed = item.get('content_last_modified')
-                        # Sanity check
-                        existing_record.enterprise_customer_catalog_uuid = enterprise_customer_catalog.uuid
-                        existing_record.save()
-                        create_payload[key] = existing_record
-                    elif key in items_update_keys:
-                        existing_record = items_to_update.get(key)
-                        existing_record.content_title = item.get('title')
-                        # Sanity check
-                        existing_record.enterprise_customer_catalog_uuid = enterprise_customer_catalog.uuid
-                        existing_record.save()
-                        # Intentionally setting the channel_metadata and content_last_changed
-                        # fields post-save to clarify what data has been transmitted
-                        # and so that untransmitted courses will get picked up again in subsequent runs
-                        existing_record.content_last_changed = item.get('content_last_modified')
-                        existing_record.channel_metadata = transformed_item
-                        update_payload[key] = existing_record
+                key_to_content_metadata_mapping = {
+                    get_content_metadata_item_id(item): item for item in content_metadata_items
+                }
+            for key, item in items_to_create.items():
 
+                self._sanitize_and_set_item_metadata(item, key_to_content_metadata_mapping[key], IC_CREATE_ACTION)
+                # Sanity check
+                item.enterprise_customer_catalog_uuid = enterprise_customer_catalog.uuid
+                item.save()
+                create_payload[key] = item
+            for key, item in items_to_update.items():
+                metadata_transformed_for_exec_ed = self._transform_exec_ed_content(
+                    key_to_content_metadata_mapping[key]
+                )
+                transformed_item = self._transform_item(metadata_transformed_for_exec_ed, action=IC_UPDATE_ACTION)
+
+                item.content_title = transformed_item.get('title')
+                item.enterprise_customer_catalog_uuid = enterprise_customer_catalog.uuid
+                item.save()
+                # Intentionally setting the channel_metadata and content_last_changed
```

Comment:
is there a way where we don't need to do this? i feel like we shouldn't encode the sync status into the presence or absence of the `content_last_changed` field - now that we have a more robust sync status on the audit records, can we just always set this field?